### ISSUE TITLE
Add "Configure Prototype Token" to the Right-Click menu in the Actor's sidebar

### ISSUE THREAD

**Issue by alaustin0** [foundryvtt/foundryvtt#11797]
### User Experience

Currenty, the only way to access an Actor's Prototype Token settings is my opening the character sheet and clicking a button in the sheet header. Being able to access this through the sidebar would be a fairly unobstrusive improvement to the UX for editing tokens.

![prototype-token-mockup](https://github.com/user-attachments/assets/4d8a499c-9abe-408d-b625-e4494b8821fa)

**Comment by aaclayton** [comment]
Makes sense. Tagging to https://github.com/foundryvtt/foundryvtt/issues/11320
